source/code/plugin/in_kube_health.rb

```diff
@@ -0,0 +1,303 @@
+#!/usr/local/bin/ruby
+# frozen_string_literal: true
+
+require_relative "KubernetesApiClient"
+require_relative "oms_common"
+require_relative "omslog"
+require_relative "ApplicationInsightsUtility"
+
+module Fluent
+
+  Dir[File.join(__dir__, './health', '*.rb')].each { |file| require file }
```

Comment:
why this ? It doesn't recognize plugins under plugins/health/ ?

Reply:
I believe ruby ONLY looks at the current folder and tries to load them. any require_relative in the same folder is redundant. but we use it anyways. Since the library files are in a sub-folder this is explicitly required. thats why all the library files dont have this require, but the plugin files do